Quizzler-iOS13/AppDelegate.swift

```diff
@@ -10,31 +10,32 @@ import UIKit
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
+    
+    var window: UIWindow?
 
     //Version 1
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let viewController = storyboard.instantiateViewController(identifier: "ViewController") as! ViewController
```

Comment:
As we've discussed, we should avoid to use force unwrap
In this specific case it should be easy, as
- storyboard.instantiateViewController returns non-optional UIViewController
- we cast here to ViewController type, but we actually do not use it ViewController properties/functions anyhow

So you can just drop "as! ViewController" part and it should also work

Reply:
In the commit "[8ada50a](https://github.com/ronaldngounou/Quizzler-iOS13/pull/2/commits/8ada50a432ddafffaecd5b940f196579bc7cc1e1)" I deleted force unwrap. The old version was not launching. In this new version, the code is running but the applications stays on a black screen. The error says: "Failed to instantiate the default view controller for UIMainStoryboardFile 'Main' - perhaps the designated entry point is not set?"

To solve this issue, I tried added the 'Main' value for the StoryBoard property in the Info.plist.

Still not working.